### DISCUSSION TITLE
A word on content and question to you all

### DISCUSSION THREAD

**Discussion by DeeJayTC** [WeAreDotnet/dotnet-creator-cards#70] [Polls]
Hey everyone!
Over the last few days, since we have our first beta online, we got approached by some asking about the content we offer, especially on the landing page. I want to take a minute to talk about that in more detail. 

First of all, we are not producing content ourselves but give everyone who does a unique and .NET-focused platform. Every content on the page is done by creators we accepted to be on the page and while we're screening everyone we might not see everything they have. 

That said, currently, there is content that is not completely .NET related, such s "how to be a better boss" or other more social /business-related topics. This content is produced by people who have a ton of .NET-related content as well. 

We're a bit torn about whether we should allow that but lean towards only allowing that on the creator's profile page and not on the landing page or future content page (we're adding a big page to browse all content across all creators!) 

Ultimately we want this to be a .NET + friends platform what I mean by that is that we allow any form of content that's directly .NET related (such as ASP, Entity framework, Blazor etc etc etc) but also content any .NET dev could be interested in such as Azure hosting, CI/CD, Github topics if they are using .NET as examples or are somehow related to .NET.  For example, we would not want an article about "how to host angular app on azure" that's just not related to us and wouldn't fit the platform. But if someone has a video about how to host a .NET app on AWS or something similar...yea totally up for that! Also something like Visual Studio Code extensibility is totally something we'd add even if its javascript based just because VSCode is a huge part of .NET development as well. I guess you get the gist!

We do however all content from anyone! No matter if you just started or wrote your 1000th blog post, we don't care and anyone who produces .NET content is totally welcome. 

We'd still love to hear from you tho, should we allow non-.NET content (definitely not other technologies but rather social topics from .NET authors!) or do you think we should completely and only focus on .NET?  

Just join the poll and also leave a message, happy to hear your opinions on that!

Tim

**Comment by danpdc** [comment]
I also think it's not useful for the broader community. I'd assume people that join as members are primarily interested in content around .NET. It would be a pitty if it becomes a place similar to other media where people gather (Twitter, Linkedin, Mastodon). For that mixed content there are plenty of options out there. But for targeted and specific .NET content there's nothing out there and you guys would totally fill that void.

**Reply by DeeJayTC** [reply]
This is the way :) 

Thats our plan and why we created the page, there's too many mixed channels and especially smaller, younger creator's are struggling there. We want to give everyone a platform no matter the age, seniority, reach etc.  And yes...focussed on .NET content only.

**Comment by TechWatching** [comment]
What do you mean exactly by allow or not allow? Will you filter the content automatically so that there is only .NET content available. Or will the creators be asked to only publish .NET content? As a "we are dotnet" member I want to only read content focused on .NET or related to .NET (CI/CD, Azure, ... as you mentioned). But as a content creator, even if my main focus is on dotnet, I write on my blog articles about things I like or want to share and it's not only about .NET. Should I have a dedicated RSS feed with only dotnet content (that's something that could be done I think)?

**Reply by DeeJayTC** [reply]
First of all we allow anyone who does produce .NET content to the platform wether or not there's other content on the blog, channel etc. 
We already asked some folks to give us feeds pre-filtered to their .NET content as often creators themselves can do that easier than  we could. 

On your own profile page we'll always allow all your content if the majority of your content is .NET + friends related. 

However we wouldn't show non related content of yours on the landing page or our more general content pages (place where you can search for  content across all authors)

So you can totally write on your blog about whatever you want as long as  you can give us a filtered feed or help us otherwise to identify your .NET content. 

And yea totally agree WeAreDotnet is about .NET content as that's whats missing and we'll always make sure to focus on that first. (I even removed 2 guys from  the landing page content already as they where mostly talking about leadership and other things)

**Reply by andychiare** [reply]
I agree with @TechWatching, and I like the approach of allowing all content in the creator's profile, but only .NET content on the landing page.
However, it is not clear to me whether this means that creators should provide two feeds: one .NET-filtered and one unfiltered 🤔

**Reply by CamiloTerevinto** [reply]
I agree, this makes sense. I imagine that for some of us it's going to be easy to provide 2 RSS feeds, our general and a dotnet one. For the rest, I imagine that plenty techniques could be used to determine whether the content is related to dotnet or not (such as keywords and what not)

**Comment by berviantoleo** [comment]
I agree with focusing on .NET content only.

In my case, I can't provide a pre-filtered feed because I use a platform that does not provide the ability to filter my post. 

I think it will be good to filter some posts and check if the post is relevant to .NET, for example, filter by the post tag. I always use tags to mark the topics, but maybe it won't work with others.

**Comment by DeeJayTC** [comment]
The main problem for us is that it's really hard to filter the content...i mean filter based on what?

 We could define a list of  things the title needs to contain to be relevant would be quite an extensive list but well. 
 Like as long as the title contains .NET, C# etc etc etc its fine. 
 
 We can already see there's stuff we probably would match. There's for example one blog post about domain models and some architecture things. The title and description wouldn't tell me its .NET related but he's making extensive use of C# based code samples in the article and a ton is about .NET. 
 
 We do not have access to the full content of the posts (and especially videos)  quite often so its tough for us to filter things on our end after all. 
 
 More than happy about any pre-filtered feed as that really makes our live a ton easier. If not possible we'd have to see how we can help  you get a filtered feed.
